src/test/kotlin/org/rust/ide/inspections/typecheck/ConvertToTyWithDerefsRefsFixTest.kt

```diff
@@ -89,40 +88,40 @@ class ConvertToTyWithDerefsRefsFixTest : RsInspectionsTestBase(RsTypeCheckInspec
         }
     """)
 
-    fun `test &T to &mut T`() = checkFixIsUnavailable("Convert to &mut i32 using dereferences and/or references", """
+    fun `test &T to &mut T`() = checkFixIsUnavailable("Convert to &mut i32", """
         fn main () {
             let a: &i32 = &42;
             let b: &mut i32 = <error>a<caret></error>;
         }
     """)
 
-    fun `test mut &T to &mut T`() = checkFixIsUnavailable("Convert to &mut i32 using dereferences and/or references", """
+    fun `test mut &T to &mut T`() = checkFixIsUnavailable("Convert to &mut i32", """
         fn main () {
             let mut a: &i32 = &42;
             let b: &mut i32 = <error>a<caret></error>;
         }
     """)
 
-    fun `test &mut&&mut T to &mut T`() = checkFixIsUnavailable("Convert to &mut i32 using dereferences and/or references", """
+    fun `test &mut&&mut T to &mut T`() = checkFixIsUnavailable("Convert to &mut i32", """
         fn main () {
             let a: &i32 = &42;
             let b: &mut i32 = <error>a<caret></error>;
         }
     """)
 
-    fun `test &mut&&mut T to &mut& T `() = checkFixByText("Convert to &mut &i32 using dereferences and/or references", """
+    fun `test &mut&&mut T to &mut& T `() = checkFixByText("Convert to &mut &i32 by applying &mut & ***", """
```

Comment:
Do we really need space between `& ***`? Code after quick fix applying doesn't contain it